flink-core/src/main/java/org/apache/flink/core/state/StateFutureUtils.java

```diff
@@ -34,6 +35,14 @@
  */
 @Experimental
 public class StateFutureUtils {
+    /**
+     * Returns a function that always returns a completed future that does nothing and return null.
+     */
+    public static <T>
+            FunctionWithException<T, StateFuture<T>, Exception> createVoidReturnFunction() {
```

Comment:
```suggestion
            FunctionWithException<T, StateFuture<Void>, Exception> returnVoidFuture() {
```

Reply:
This util function is no use to me anymore, and I have removed it.